### ISSUE TITLE
Games.com noit appearing

### ISSUE THREAD

**Issue by MattyH81** [rg35xx-cfw/rg35xx-cfw.github.io#221]
Tried adding roms both as .zip and .bin.  system does not show in menu.

**Comment by uzumak11** [comment]
Which system are you referring to?

**Comment by MattyH81** [comment]
It's in the title.  Games.com (tiger)

**Comment by uzumak11** [comment]
https://wiki.batocera.org/systems:gamecom

Not sure about support on our device though. 

It's not part of the release notes under "Cores/Emulators status"

**Comment by MattyH81** [comment]
Yeah. I saw the folder had been added do thought I'd try.  I'll leave roms for there now in case it gets added.

**Comment by uzumak11** [comment]
The folders are part of batocera main (linux). 

The release notes are a better indicator of what is currently supported, so be sure to check that.

Having said that, the dev is planning to merge our device build with batocera main rather soon, which would hopefully add support for all the extra systems.

I should check out this system though, never heard of it growing up!
